### DISCUSSION TITLE
DLC for linux

### DISCUSSION THREAD

**Discussion by Islaifer** [Russifiers-for-Humans/Europa-Universalis-4-DLC#1] [Q&A]
Hi, I'm using linux, and I want to use the crack for unlock all dlcs, have one method to run this crack on linux?

**Comment by 55TeMeR55** [comment]
Hello, everything is more complicated with Linux, only proton (Windows emulator on Linux) immediately comes to mind and there is also a forum, I would use the emulator

https://www.strategium.ru/forum/topic/94894-dlc-дополнения-к-eu4-для-linux/page/2/
